oscilloscope/oscilloscope_server.py

```diff
@@ -0,0 +1,227 @@
+"""
```

Comment:
Consider calling this file just `server.py`. The module name `oscilloscope.server` is unambiguous and less repetitive.